packages/imhex.vm/imhex.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>imhex.vm</id>
+    <version>1.31.0.20231018</version>
```

Comment:
[nitpick] The version is the tool version, so `1.31.0` is enough. The forth segment only needs to be used if we update the package without increasing the tool version, as the package version always needs to be increased. See some the [documentation in the Wiki](https://github.com/mandiant/VM-Packages/wiki/Package-Structure#version) and let me know if you think it is not clear.

Reply:
Resolved at https://github.com/mandiant/VM-Packages/pull/700/commits/a0a79e9ca926a7158a5e9f988ecc35b328e43dab.